src/wrapping.rs

```diff
@@ -1,11 +1,11 @@
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum WrappingMode {
     Character,
-    NoWrapping,
+    NoWrapping(bool), // explicitly opted in or not
```

Comment:
Maybe: "The bool specifies whether wrapping has been explicitly disabled by the user via `--wrap=never`"?

Reply:
I like that. Moving to a line above as it is a little easier to read that way.